scripts/release.sh

```diff
@@ -111,6 +111,7 @@ function check_tag_correct() {
 
     # If the tapd reported version contains a suffix, remove it, so we can match
     # the tag properly.
+    # shellcheck disable=SC2001
```

Comment:
Should this new line be included in this PR?

Reply:
Yes, shellcheck (the linter I used to find these issues) flags this sed expression but I don't think we want to replace it with something bash-specific / less portable.